chia/types/fee_rate.py

```diff
@@ -0,0 +1,10 @@
+from dataclasses import dataclass
+
+from chia.util.ints import uint64
+
+
+@dataclass
+class FeeRate:
```

Comment:
are we using this somewhere ?

Reply:
Yes, this is another one of those types that are just there to prevent a raw uint64 being interpreted in a way the programmer might not have intended. I think this one is probably the most important, though. There's a HUGE difference between a "fee estimate" (mojos) and a FeeRate estimate (mojos per cost)